steps/containers_kube.py

```diff
@@ -9,6 +9,13 @@ def step_impl(context, image, host):
                                host,
                                module_args='docker pull %s' % image)
 
+@then('docker pull "{image}"')
+def step_impl(context, image):
+    '''docker pull image'''
+    assert context.remote_cmd('command',
+                               remote_user='root',
```

Comment:
`remote_user` is an ansible API arg. This is a hack until we add support for this in the dynamic inventory script. Static inventory supports this so `remote_user` wouldn't be necessary.